app/jsx/components/BreadcrumbComp.jsx

```diff
@@ -5,21 +5,24 @@ import { Link } from 'react-router'
 
 class BreadcrumbComp extends React.Component {
   render() {
-    var lastN = this.props.array.length - 1
-
-    var nodes = this.props.array.map(function(node, i) {
-      if (node.id == 'root') {
-        return (
-          <Link className={(i==lastN) ? "c-breadcrumb-link--active": null} to="/" key={i}>{node.name}</Link>
-        )
-      }
-      return (
-        <Link className={(i==lastN) ? "c-breadcrumb-link--active": null} to={"/unit/" + node.id} key={i}>{node.name}</Link>
-      )
-    });
+    let missingNavStyle = this.props.array ? null : { margin: '20px 0px 20px' },
```

Comment:
I like the workaround here. I'd suggest adding your comment here as a comment in the code as well, but that won't stop me from merging this PR.